src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -212,27 +211,27 @@ export default {
             parameters['union'] = union;
             return parameters
         },
-        getAnnotations(label, filters, union) {
+        getAnnotations(label) {
             let promise1;
             let promise2;
 
-            let labelFilters = {label: label.id};
-            if (filters.length > 0){
-                labelFilters['filters'] = filters;
-                labelFilters['union'] = union;
-            }
+            let cacheKey = JSON.stringify({label: label.id});
```

Comment:
Better revert this to the previous behavior which just used the label ID as cache key. No need to stringify a whole object.